docs/learn/learn-governance.md

```diff
@@ -232,20 +271,33 @@ the result. `Super-Majority Approve` requires more `aye` votes to pass the refer
 is low, therefore, based on the above result, the referendum will be rejected. In addition, only the
 winning voter's tokens are locked. If the voters on the losing side of the referendum believe that
 the outcome will have negative effects, their tokens are transferrable so they will not be locked
-into the decision. Moreover, winning proposals are autonomously enacted only after some enactment
-period.
+into the decision. Moreover, winning proposals are autonomously enacted only after the enactment delay period.
+
+#### Voting on a referendum (governance v2)
+
+In governance v2, a proposal is approved if it meets the requirements for **approval** and **support**, removing the adaptive quorum biasing system. 
+
+Approval is defined as the share of approval vote-weight (after adjustment for conviction) against the total number of vote-weight (for both approval and rejection).
+
+Support is the total number of votes in the approval (ignoring any adjustment for conviction) compared to the total possible amount of votes that could be made in the system.
+
+It must fulfill this criteria for the minimum of the **Confirmation Period**. Different tracks have different Confirmation Periods and requirements for approval and support. It is now possible to configure a proposal in such a way that an increasingly lower amount of support and overall approval for the proposal are needed for it to pass. With proposals that use less privileged origins, it is far more reasonable to drop the required turnout to a more realistic amount earlier than those which use highly privileged classes such as `Root`. Similarly, classes which command more political significance will tend to accept less controversy (and thus require a higher approval) early on.
```

Comment:
```suggestion
It must fulfill this criteria for the minimum of the **Confirmation Period**. Different tracks have different Confirmation Periods and requirements for approval and support. It is now possible to configure the amount of support and overall approval required for it to pass. With proposals that use less privileged origins, it is far more reasonable to drop the required turnout to a more realistic amount earlier than those which use highly privileged classes such as `Root`. Classes with more political significance can be made to require a higher approval early on, to avoid controversy.
```